rmf_site_format/src/misc.rs

```diff
@@ -353,6 +353,33 @@ impl Default for Pose {
     }
 }
 
+#[cfg(feature = "urdf")]
+impl From<Pose> for urdf_rs::Pose {
+    fn from(pose: Pose) -> Self {
+        urdf_rs::Pose {
+            rpy: match pose.rot {
+                Rotation::EulerExtrinsicXYZ(arr) => urdf_rs::Vec3(arr.map(|v| v.radians().into())),
+                Rotation::Yaw(v) => urdf_rs::Vec3([0.0, 0.0, v.radians().into()]),
+                Rotation::Quat([x, y, z, w]) => {
+                    let (z, y, x) = glam::quat(x, y, z, w).to_euler(glam::EulerRot::ZYX);
+                    urdf_rs::Vec3([x as f64, y as f64, z as f64])
+                }
+            },
+            xyz: urdf_rs::Vec3(pose.trans.map(|v| v as f64)),
+        }
+    }
+}
+
+#[cfg(feature = "urdf")]
+impl From<&urdf_rs::Pose> for Pose {
+    fn from(pose: &urdf_rs::Pose) -> Self {
+        Pose {
+            trans: pose.xyz.map(|t| t as f32),
+            rot: Rotation::EulerExtrinsicXYZ(pose.rpy.map(|t| Angle::Rad(t as f32))),
+        }
+    }
+}
+
```

Comment:
I believe these have to be implemented here, I added a default off `urdf` feature to remove the dependency unless needed.